apps/meteor/client/views/admin/rooms/RoomsTableFilters.tsx

```diff
@@ -78,21 +92,22 @@ const RoomsTableFilters = ({ setFilters }: { setFilters: Dispatch<SetStateAction
 		>
 			<Box minWidth='x224' display='flex' m='x4' flexGrow={2}>
 				<TextInput
+					data-qa-id='AdminRoomSearchInput'
 					name='search-rooms'
 					alignItems='center'
 					placeholder={t('Search_rooms')}
 					addon={<Icon name='magnifier' size='x20' />}
 					onChange={handleSearchTextChange}
-					value={text}
+					value={prevRoomFilters.current.searchText}
 				/>
 			</Box>
-			<Box minWidth='x224' m='x4'>
+			<Box minWidth='x224' m='x4' data-qa-id='AdminRoomDropdownInput'>
```

Comment:
It would be good to use roles in this case, maybe @dougfabris could help us on the best way to do it here

Reply:
yeah, first thing I can see, this select is not navigable by keyboard at all. I would suggest adding at least a button role on the anchor and adding the proper button keyboard event. With the role I believe we can get the locator using `locator.getByRole` instead of the custom `data-qa-id` prop